sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuTransitionOverrides.scala

```diff
@@ -574,6 +565,18 @@ object GpuTransitionOverrides {
       case _ => plan
     }
   }
+
+  /**
+   * Check the Expression is or has Input expressions.
+   * @param exec expression to check
+   * @return true or false
+   */
+  def checkHasInputExpressions(exec: Expression): Boolean = exec match {
```

Comment:
nit: can we rename this `checkHasInputFileExpressions`

Reply:
Done